src/routes/boot.ts

```diff
@@ -359,9 +363,11 @@ const anonymousBoot = async (
     middleware ? middleware(con, req, res) : {},
     getAnonymousFirstVisit(req.trackingId),
   ]);
+  const isPreOnboardingV2 = new Date(firstVisit) < onboardingV2Requirement;
   return {
     user: {
       firstVisit,
+      isPreOnboardingV2,
```

Comment:
Just a suggestion why dont you immediately change the flag value in api?
We have done this before so from the frontend you'll only have to rely on flags

Reply:
Oh, never knew we've done that in the past, but we can definitely do that to make it easier.